src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int indexOfColor = new Random().nextInt(Color.values().length);
+        Color color = Color.values()[indexOfColor];
+        String getColor = color.toString();
+        return getColor;
```

Comment:
1. Please make Random class variable in order not to create new object each time method is called
2. String representation of enum should be called with .name() method
```suggestion
        int indexOfColor = random.nextInt(Color.values().length);
        return Color.values()[indexOfColor].name();
```